.github/workflows/publish-sdist-wheels.yml

```diff
@@ -25,69 +27,70 @@ jobs:
         with:
           submodules: 'true'
 
-      - uses: actions/setup-python@v4
-        name: Install Python
+      - name: Install Python
+        uses: actions/setup-python@v4
         with:
-          python-version: '3.7'
```

Comment:
Why not just jump to 3.10 so that we don't have to change it soon?

Reply:
good call